.gitignore

```diff
@@ -1 +1,2 @@
-TODO.md
\ No newline at end of file
+TODO.md
```

Comment:
Pull the master branch down and resolve this merge conflict locally then push back up to this branch.